Conda-forge's build environment on ARM uses sysroot_linux-aarch64 2.17, which
is based on glibc 2.17 so <sys/auxv.h> does not define HWCAP_* values on ARM.
Work around this by including the kernel header to get the desired values.

See https://github.com/samtools/htscodecs/issues/88
---
diff --git a/src/htslib-1.18/htscodecs/htscodecs/rANS_static4x16pr.c b/src/htslib-1.18/htscodecs/htscodecs/rANS_static4x16pr.c
index 732d9f5..0592a78 100644
--- a/src/htslib-1.18/htscodecs/htscodecs/rANS_static4x16pr.c
+++ b/src/htslib-1.18/htscodecs/htscodecs/rANS_static4x16pr.c
@@ -1011,6 +1011,8 @@ unsigned char *(*rans_dec_func(int do_simd, int order))
 
 #if defined(__linux__) || defined(__FreeBSD__)
 #include <sys/auxv.h>
+// Ensure ARM HWCAP_* values are defined even on old glibc
+#include <asm/hwcap.h>
 #elif defined(_WIN32)
 #include <processthreadsapi.h>
 #endif

